ambry-protocol/src/main/java/com.github.ambry.protocol/MessageInfoListSerde.java

```diff
@@ -31,41 +31,52 @@
   private final List<MessageInfo> messageInfoList;
   static final short MessageInfoListVersion_V1 = 1;
   static final short MessageInfoListVersion_V2 = 2;
+  static final short MessageInfoListVersion_V3 = 3;
```

Comment:
minor: A lot of these constants in ambry-protocol are capitalized in weird ways. Could we start changing them to CAPITAL_SNAKE_CASE to make them uniform with other parts of the codebase